index.css

```diff
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Bebas+Neue&family=IBM+Plex+Serif:wght@300&family=Lora&family=Merriweather:wght@300&family=Montserrat:wght@300&family=Roboto:wght@300&display=swap");
+
+.font {
```

Comment:
You can remove the other fonts as only the last one is used

Reply:
I didn't know this..Thanks!!